### ISSUE TITLE
Improve accessibility of popups in Fassungsansicht

### ISSUE THREAD

**Issue by vvvyyynet** [DHBern/presentation_parzival#99]
Tab-Cycling through the popup-triggers in Fassungsansicht will trigger a scroll as well as a data-reload... hence we are stuck in the same column and the other columns cannot be reached with the keyboard, which is bad for accessibility.

Can the cycling-logic somehow be nested?

**Comment by flicksolutions** [comment]
Users of Screenreaders can navigate our site relatively simple by jumping from heading to heading (in NVDA it's the shortcut _H_) So i consider this done. ok for you @vvvyyynet ?
